maxim/logger/crewai/client.py

```
@@ -646,25 +699,24 @@ def maxim_wrapper(self, *args, **kwargs):
                                 "tags": {"span_id": span.id},
                             }
                         )
+                        setattr(tool_call, "_input", processed_inputs["query"])
```

Ruff (0.12.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)